Paylike/Adapter.php

```diff
@@ -0,0 +1,67 @@
+<?php
+namespace Paylike;
+/**
+ * Class Adapter
+ * @package Paylike
+ * The adapter class taking care of the calls to the api.
+ *
+ * The purpose of this is to abstract the requests
+ * so that this can be changed depending on the environment.
+ *
+ * @version    1.0.0
+ */
+class Adapter {
+
+	public $api_url = 'https://api.paylike.io';
+	private $api_key;
```

Comment:
I think we should stick to either doing camel case or snake case, not both (see e.g. `setApiKey`). I am in favour of camel case myself (in JavaScript), but whatever is most natural for PHP would do.